app/src/main/java/com/techlads/composetv/features/home/leftmenu/data/MenuData.kt

```diff
@@ -25,6 +25,14 @@ object MenuData {
         MenuItem(Favorites.title, "Favorites", LineAwesomeIcons.HeartSolid),
     )
 
+    val topBarMenuItems = listOf(
```

Comment:
@SeptAlfauzan I will request to use the existing menu items because it will be same navigation different composable :)

Reply:
okay I will fix it, I thought because in sample design the placement of setting is in the end of list I thougt it will okay to create another menu item